inclusion_connect/oidc_overrides/views.py

```diff
@@ -81,6 +84,11 @@ def create_authorization_response(self, request, scopes, credentials, allow):
             application=application,
             defaults={"last_login": timezone.now()},
         )
+
+        # Clear session. Don't pop so that we only only update the session expire date if the entry exists.
```

Comment:
```suggestion
        # Clear session. Don't pop(), only update the session expiry if the entry exists.
```

Je me demande si un test serait utile. Pourquoi ne pas mettre à jour la session ?

Reply:
Je veux que la session ne dure que 30 minutes après le login donc éviter qu'à chaque appel a /authorize par un partenaire la session ne soit rallongée.

Reply:
Oui, donc un test serait utile à mon avis.

Reply:
Il existe déjà (test_session_duration) c'est là où tu as mis un commentaire.

Reply:
Il ne casse pas si jamais on remplace les deux lignes par `self.request.session.pop(OIDC_SESSION_KEY, None)`.

Reply:
:facepalm: en effet....